generators/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- code-approvers
```

Comment:
How are `code-approvers` different than `approvers`, same for `code-reviewers` and `reviewers`? why do we need both?

Reply:
`code-approvers` is the group of people that are maintaining the automation code in this repository. Since I couldn't imagine a better name, I stuck to what I found in k/kubevirt, from which this comes - if you insist I could rename it to i.e. `automation-code-reviewers` and `automation-code-approvers` or do you have a better suggestion?

Reply:
I prefer `automation-code-*` as it is more descriptive.
But I'm also perfectly content to not change it since this matches k/kubevirt and the entry has a descriptive note.